ClosedXML/Excel/Drawings/PictureEnums.cs

```diff
@@ -8,15 +8,16 @@ public enum XLMarkerPosition
 
     public enum XLPictureFormat
     {
-        Bmp = 0,
-        Gif = 1,
-        Png = 2,
-        Tiff = 3,
-        Icon = 4,
-        Pcx = 5,
-        Jpeg = 6
-        //Emf = 7,
-        //Wmf = 8
+        Unknown,
+        Bmp,
+        Gif,
+        Png,
+        Tiff,
+        Icon,
+        Pcx,
+        Jpeg,
+        Emf,
+        Wmf
```

Comment:
Technically, this is a breaking change, though the obstacles in which this may have effect are quite exotic.

Reply:
I'm happy to accept that risk. I highly doubt it will affect anyone.

Reply:
I do not object either, just pointed that maybe we should put a label on this. May not worth it though.